setup.py

```diff
@@ -41,6 +41,7 @@
         "Programming Language :: Python :: 3.7",
```

Comment:
Spyder 6 is not going to be compatible with Python 3.7 because by the time it's released 3.7 will have reached its end-of-life date.